dwave/system/samplers/dwave_sampler.py

```diff
@@ -522,3 +524,221 @@ def to_networkx_graph(self):
+            ~1 ms pause on a D-Wave system.
+
+            >>> from dwave.system import DWaveSampler, FixedEmbeddingComposite
+            >>> from minorminer import find_embedding
+            >>> import dimod
+            >>> import random
+            >>> import networkx as nx
+            ...
+            >>> G = nx.generators.complete_graph(100)
+            >>> bqm = dimod.from_networkx_graph(G, "BINARY")
+            >>> bqm.add_linear_from({node: random.choices([-0.5, 0.5])[0] for node in bqm.linear.keys()})
+            >>> bqm.add_quadratic_from({edge: random.choices([-1, 1])[0] for edge in bqm.quadratic.keys()})
+            ...
+            >>> qpu = DWaveSampler()
+            >>> embedding = find_embedding(bqm.quadratic.keys(), qpu.edgelist)  # doctest: +SKIP
+            >>> num_qubits = sum(len(chain) for chain in embedding.values())    # doctest: +SKIP
+            ...
+            >>> reads = 500
+            >>> anneal_schedule = [[0.0, 0.0], [40.0, 0.4], [1040.0, 0.4], [1042, 1.0]]
+            >>> estimated_runtime = qpu.estimate_qpu_access_time(num_qubits,
+            ...                          num_reads=reads,
+            ...                          anneal_schedule=anneal_schedule)    # doctest: +SKIP
+            >>> print("%.1f" % estimated_runtime)  # doctest: +SKIP
+            83952.1
+
+        """
+        if anneal_schedule and annealing_time:
+            raise ValueError("Set only one of ``anneal_schedule`` or ``annealing_time``")
+
+        try:
+            problem_timing_data = self.properties['problem_timing_data']
+
+            version_timing_model = problem_timing_data['version']
+            typical_programming_time = problem_timing_data['typical_programming_time']
+            ra_with_reinit_prog_time_delta = problem_timing_data['reverse_annealing_with_reinit_prog_time_delta']
+            ra_without_reinit_prog_time_delta = problem_timing_data['reverse_annealing_without_reinit_prog_time_delta']
+            default_programming_thermalization = problem_timing_data['default_programming_thermalization']
+            default_annealing_time = problem_timing_data['default_annealing_time']
+            readout_time_model = problem_timing_data['readout_time_model']
+            readout_time_model_parameters = problem_timing_data['readout_time_model_parameters']
+            qpu_delay_time_per_sample = problem_timing_data['qpu_delay_time_per_sample']
+            ra_with_reinit_delay_time_delta = problem_timing_data['reverse_annealing_with_reinit_delay_time_delta']
+            ra_without_reinit_delay_time_delta = problem_timing_data['reverse_annealing_without_reinit_delay_time_delta']
+            decorrelation_max_nominal_anneal_time = problem_timing_data['decorrelation_max_nominal_anneal_time']
+            decorrelation_time_range = problem_timing_data['decorrelation_time_range']
+
+            default_readout_thermalization = self.properties['default_readout_thermalization']
```

Comment:
This seems odd. "default_readout_thermalization" is in the problem_timing_data too, isn't it?

Reply:
it is, and I can take it from there

Reply:
Done. Also added missing return value in docstring.